src/rpc/mining.cpp

```diff
@@ -849,18 +887,22 @@ UniValue getblocksubsidy(const UniValue& params, bool fHelp)
         throw runtime_error(
             "getblocksubsidy height\n"
             "\nReturns block subsidy reward, taking into account the mining slow start and the community fund, of block at index provided.\n"
+            
             "\nArguments:\n"
-            "1. height         (numeric, optional) The block height.  If not provided, defaults to the current height of the chain.\n"
+            "1. height                    (numeric, optional) the block height\n"
+            "                              if not provided, defaults to the current height of the chain\n"
+            
             "\nResult:\n"
             "{\n"
-            "  \"miner\" : x.xxx           (numeric) The mining reward amount in ZEN.\n"
-            "  \"community\" : x.xxx        (numeric) The community fund amount in ZEN.\n"
-            "  \"securenodes\" : x.xxx        (numeric) The securenodes fund amount in ZEN (only if securenode HF fund is available).\n"
-            "  \"supernodes\" : x.xxx        (numeric) The supernodes fund amount in ZEN (only if supernode HF fund is available).\n"
+            "  \"miner\": xxxx,           (numeric) the mining reward amount in " + CURRENCY_UNIT + "\n"
+            "  \"community\": xxxx,       (numeric) the community fund amount in " + CURRENCY_UNIT + "\n"
+            "  \"securenodes\": xxxx,     (numeric) the securenodes fund amount in " + CURRENCY_UNIT + " (only if securenode HF fund is available)\n"
+            "  \"supernodes\": xxxx       (numeric) the supernodes fund amount in " + CURRENCY_UNIT + " (only if supernode HF fund is available)\n"
```

Comment:
```suggestion
            "  \"securenodes\": xxxx,     (numeric) the securenodes fund amount in " + CURRENCY_UNIT + "\n"
            "  \"supernodes\": xxxx       (numeric) the supernodes fund amount in " + CURRENCY_UNIT + "\n"
```
As the fork is active now this can be removed.